flow-libdef/@ginkgo-bioworks/react-json-schema-form-builder_v1.x.x/test_react-json-schema-form-builder_v1.x.x.js

```diff
@@ -20,7 +21,54 @@ describe('@ginkgo-bioworks/react-json-schema-form-builder', () => {
     mods: {},
     className: 'foo'
   };
-
+  const propsWithMods = {
```

Comment:
You're not using this anywhere yet -- is it just for example purposes?

Reply:
aah oops, I meant to include it in a test below.